src/android/MyFirebaseMessagingService.java

```diff
@@ -42,6 +42,8 @@ public void onMessageReceived(RemoteMessage remoteMessage) {
 		
 		Map<String, Object> data = new HashMap<String, Object>();
 		data.put("wasTapped", false);
+		data.put("title", remoteMessage.getNotification().getTitle());
```

Comment:
You should add a null check for " remoteMessage.getNotification()", because you send only data in FCM API , then background notification works - explained [here](https://firebase.google.com/docs/cloud-messaging/concept-options#data_messages). So in that case, 'remoteMessage.getNotification()' will be null and it will break.